app/connectors/SubscriptionConnector.scala

```diff
@@ -85,7 +84,18 @@ class SubscriptionConnector @Inject() (val config: FrontendAppConfig, val http:
     }
   }
 
-  def handleError[A](responseMessage: HttpResponse): Either[ApiError, A] =
+  def updateSubscription(requestDetail: UpdateSubscriptionRequest)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
+    val updateSubscriptionUrl = s"${config.businessMatchingUrl}/subscription/update-subscription"
+    http
+      .PUT[UpdateSubscriptionRequest, HttpResponse](updateSubscriptionUrl, requestDetail)
+      .map {
+        responseMessage =>
+          logger.warn(s"updateSubscription: Status ${responseMessage.status} has been received when update subscription was called")
```

Comment:
I think this should this be an `info` log rather than a `warn` log. It's getting logged on every response - successes and failures.